SUMMARY.md

```diff
@@ -1,19 +1,20 @@
-# Table of contents
-
-### Documents
-* [Mojaloop Specification Overview](README.md)
-  * [Glossary](fspiop-api/documents/Glossary.md)
-  * [Logical data Model](fspiop-api/documents/Logical Data Model.md)
-  * [Generic Transaction Patterns](fspiop-api/documents/Generic Transaction Patterns.md)
-  * [Use Cases](fspiop-api/documents/Use Cases.md)
-  * [API Definition v1.1](fspiop-api/documents/API Definition v1.1.md)
-  * [Scheme Rules](fspiop-api/documents/Scheme Rules.md)
-  * [JSON Binding Rules](fspiop-api/documents/JSON Binding Rules.md)
-  * [Scheme Rules](fspiop-api/documents/Scheme Rules.md)
-  * [PKI Best Practices](fspiop-api/documents/PKI Best Practices.md)
-  * [Signature v1.1](fspiop-api/documents/Signature v1.1.md)
-  * [Encryption v1.1](fspiop-api/documents/Encryption v1.1.md)
-### Change Control Board
-* [Change Control Board Overview](ccb-meetings/README.md)
-  * [Issue-and-Decision-Log](ccb-meetings/Issue-and-Decision-Log.md)
-<!--* [Changelog](changelog.md)-->
+# Table of contents
+
+### Documents
+* [Mojaloop Specification Overview](README.md)
+  * [Glossary](fspiop-api/documents/Glossary.md)
+  * [Logical data Model](fspiop-api/documents/Logical Data Model.md)
+  * [Generic Transaction Patterns](fspiop-api/documents/Generic Transaction Patterns.md)
+  * [Use Cases](fspiop-api/documents/Use Cases.md)
+  * [API Definition v1.1](fspiop-api/documents/API-Definition_v1.1.md)
+  * [Central Ledger API v1.0](admin-api/admin-api-specification-v1.0.md)
+  * [Scheme Rules](fspiop-api/documents/Scheme Rules.md)
+  * [JSON Binding Rules](fspiop-api/documents/JSON Binding Rules.md)
+  * [Scheme Rules](fspiop-api/documents/Scheme Rules.md)
+  * [PKI Best Practices](fspiop-api/documents/PKI Best Practices.md)
+  * [Signature](fspiop-api/documents/Signature.md)
+  * [Encryption](fspiop-api/documents/Encryption.md)
+### Change Control Board
+* [Change Control Board Overview](ccb-meetings/README.md)
+  * [Issue-and-Decision-Log](ccb-meetings/Issue-and-Decision-Log.md)
+<!--* [Changelog](changelog.md)-->

```

Comment:
Let's remove this commented line or reinstate it.

Reply:
@mdebarros, please, can you tell me if the changelog is something to be removed or reinstated? Thanks.

Reply:
@vandaillyes-modusbox - I think we can remove it for now..

Reply:
Thanks, I removed it then.